### DISCUSSION TITLE
Encrypt npm token for self-hosted bitbucket - what is the correct organization?

### DISCUSSION THREAD

**Discussion by ronaldploeger** [renovatebot/renovate#12197] [Request Help] [answered]
Hi,

I stored a copy of https://app.renovatebot.com/encrypt (and replaced the public key with my own) and would like to encrypt my npm token. 

We are using bitbucket - what do I have to put into the field for organization?

Thanks & best regards,
Ronald

**Comment by HonkingGoose** [comment]
We have a documentation page on private package support [^private], which explains how to authenticate with Renovate. Did you read that page already?

[^private]: https://docs.renovatebot.com/getting-started/private-packages/

**Reply by rarkins** [reply]
Do you run with `--autodiscover` or you provide a list of repos?

If your full repo name is e.g. `foo/bar` then the organization is `foo`.

**Comment by ronaldploeger** [comment, marked as answer]
Thank you @rarkins,

You are right - the organization is what is called a project in bitbucket. 

All the best,
Ronald
